tools/dispatch_data.py

```diff
@@ -69,6 +70,7 @@ def main():
     parser.add_argument('--out-dir', type=str, help='Location of the output directory where the graph partitions will be created by this pipeline')
     parser.add_argument('--ip-config', type=str, help='File location of IP configuration for server processes')
     parser.add_argument('--master-port', type=int, default=12345, help='port used by gloo group to create randezvous point')
+    parser.add_argument('--log', type=str, default="info", help='to enable logging at a given level')
```

Comment:
`--log_level` is better, I think.

Reply:
and could we print out all supported levels in the help message?

Reply:
Done as suggested.